Simplenote/NoticeView.swift

```diff
@@ -47,20 +49,23 @@ class NoticeView: UIView {
     private func setupView() {
         translatesAutoresizingMaskIntoConstraints = false
         setupViewStyles()
-        setupLongPress()
+        setupGestureRecognizers()
 
         layoutIfNeeded()
     }
 
-    private func setupLongPress() {
+    private func setupGestureRecognizers() {
         let longPressGesture = UILongPressGestureRecognizer(target: self, action: #selector(viewWasLongPressed(_:)))
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(viewWasTapped(_:)))
```

Comment:
Does this work as well when you tap on an action button? I didn't check, but I have an impression that UIButton would intercept touches. 🤔

Reply:
I tested it.  The button intercepts the touches, so the notice does not get dismissed and the button action fires

Reply:
I would think that is expected behavior though, correct?  That the button can be tapped on without dismissing the notice, and tapping anywhere else but on the button will cause a dismiss